doc/src/Intro_authors.txt

```diff
@@ -58,322 +63,3 @@ Terry Stouch (Lexicon Pharmaceuticals, formerly at Bristol Myers Squibb)
 Steve Lustig (Dupont)
 Jim Belak and Roy Pollock (LLNL) :ul
 
-:line
```

Comment:
@sjplimp you're removing a lot of content, is this placed somewhere on the webpage?

Reply:
yes, it's all on the webpage.  The only content removed from the doc page is the long list
of contributions with dates.  Those are easier to update on the website when I post a new
patch release tarball, rather than push another PR.  The authors.html link will be correct
when the patch release is posted.